src/foundry/foundry.js/handlebarsHelpers.d.ts

```diff
@@ -119,24 +111,77 @@ declare class HandlebarsHelpers {
 }
 
 declare namespace HandlebarsHelpers {
-  interface EditorOptions {
+  interface ColorPickerOptions extends Handlebars.HelperOptions {
+    hash: {
+      /**
+       * The name of the field to create
+       */
+      name?: string;
+
+      /**
+       * The current color value
+       */
+      value?: string;
+
+      /**
+       * A default color string if a value is not provided
+       */
+      default?: string;
+    };
+  }
+
+  interface EditorOptions extends Handlebars.HelperOptions {
     hash: {
+      /**
+       * The named target data element
+       */
+      target: string;
+
+      /**
+       * Is the current user an owner of the data?
+       */
+      owner?: boolean;
+
+      /**
+      button?: boolean;
+       *
+       */
       button?: boolean;
 
+      /**
+       * Is the text editor area currently editable?
+       */
+      editable?: boolean;
+
       /**
        * @defaultValue `''`
```

Comment:
```suggestion
       * @defaultValue `true`
```

Reply:
To be honest, I'd prefer to just red rid of all of these default values. We should reserve default for actual default values in javascript functions. Here, there are local variables which are initialized to these values or a fallback value. but the parameters themselves don't have default values.

Are you OK with that @UFOMelkor?

Reply:
From a user point of view, this might be a step backwards, but from a developer point of view it will make maintaining a lot easier. Hence :+1: from me.